scipy/stats/_mstats_basic.py

```diff
@@ -2897,6 +2897,32 @@ def kurtosis(a, axis=0, fisher=True, bias=True):
     -----
     For more details about `kurtosis`, see `scipy.stats.kurtosis`.
 
+    Examples
+    --------
+    In this example, we generate an array using the `scipy.stats.norm.rvs`
```

Comment:
For the doc build failure you can see in the CI testing, I did a quick check of our previous usages of this:

`git grep -E -i "scipy.stats.norm.rvs"`

```
scipy/stats/_continuous_distns.py:    >>> data = scipy.stats.norm.rvs(size=100000, loc=0, scale=1.5,
scipy/stats/_hypotests.py:    Suppose we wish to test whether data generated by ``scipy.stats.norm.rvs``
scipy/stats/_hypotests.py:    ``scipy.stats.norm.rvs`` have the same distribution. We choose a
scipy/stats/_mstats_basic.py:    In this example, we generate an array using the `scipy.stats.norm.rvs`
```

so, it looks like an extra set of backticks may do the trick!